### DISCUSSION TITLE
[ERROR] No matching export in "node_modules/pinia/node_modules/vue-demi/lib/index.mjs" for import "hasInjectionContext"

### DISCUSSION THREAD

**Discussion by joansebgarcia** [vuejs/pinia#2334] [Help and Questions]
### Reproduction

nothing to reproduce

### Steps to reproduce the bug

Hi I am in the process of migrating from vue 2 to vue 3 I followed the step by step documentation of vue 
**https://v3-migration.vuejs.org/** then migrate from vue-cli to vite following this tutorial: 
**https://vueschool.io/articles/vuejs-tutorials/how-to-migrate-from-vue-cli-to-vite/**
 Then I migrated from vuex to pinia but I get this error

![bug](https://github.com/vuejs/pinia/assets/116913262/7b72c212-1577-4211-b0d6-16281c0ca399)
![vite_bug](https://github.com/vuejs/pinia/assets/116913262/0cc854ab-9aa7-4c91-ab31-dae597f87629)

Apparently it asks me to actuallizar the dependecias of pinia not if the version of pinia is not compatible with the version of vue that I have
![pinia_version](https://github.com/vuejs/pinia/assets/116913262/5de5973d-4bbc-46e1-a8f0-055648a4926d)

Thanks for reading

### Expected behavior

That was not happening to me until today it was already in operation

### Actual behavior

I keep getting the same problem

### Additional information

Thank you

**Comment by posva** [comment]
You need to update the version of vue-demi with `npm update vue demi`. Replace with the package manager you user of course
